README.md

```diff
@@ -133,13 +171,30 @@ minikube tunnel
 The external IP should be assigned:
 
 ```bash
-kubectl get all -n pizza-ns
+kubectl get services
 
-NAME                 TYPE           CLUSTER-IP       EXTERNAL-IP   PORT(S)          AGE
-service/pizza-info   LoadBalancer   10.108.142.101   127.0.0.1     3333:30076/TCP   39s
+NAME                  TYPE           CLUSTER-IP       EXTERNAL-IP     PORT(S)          AGE
+quickpizza-frontend   LoadBalancer   10.99.177.165    10.99.177.165   3333:30333/TCP   3m9s
 ```
 
-Now you can go to [localhost:3333](http://localhost:3333) and get some pizza recommendations!
+You should now be able to access the application on port `3333` in the IP address noted below in your browser, which in our example was `10.99.177.165`. Depending on the OS you're using, it might be `127.0.0.1`, which is also fine.
+
+We can save this IP on an environment variable for using it later on tests:
+
+```shell
+export BASE_URL="http://$(kubectl get svc quickpizza-frontend -o jsonpath='{.status.loadBalancer.ingress[0].ip}'):3333"
```

Comment:
Port forwarding is delicate as it does not interact very well with the disruptor yet: https://github.com/grafana/xk6-disruptor/issues/254

On its current shape, injecting faults on the port-forwarded service (frontend/gateway in this case) may cause port-forward to fail and require the user to restart it, which is not particularly great.

We can always add a warning explaining this limitation, but I think it might be more confusing than the tunnel approach. WDYT?

Reply:
Right, right. I wrote this before Pablo explained this in our sync.

Let's avoid adding warnings for now and focus the explanations on Minikube - where the IP approach is rather easy :+1: